docs/code_ref/io.rst

```diff
@@ -1,106 +1,26 @@
-Input/output (`sunpy.io`)
+*************************
+Input/Output (`sunpy.io`)
 *************************
 
-``sunpy.io`` contains readers for files that are commonly used in solar physics.
-
-Unified File Readers
-====================
+`sunpy.io` contains readers for files that are commonly used in solar physics.
+These readers are not designed to be used directly, they are used internally.
 
 .. automodapi:: sunpy.io
 
 .. automodapi:: sunpy.io.header
 
-.. _iofits:
+.. _io_fits:
 .. automodapi:: sunpy.io.fits
    :allowed-package-names: sunpy.io._fits
 
-.. _iojp2:
+.. _io_jp2:
 .. automodapi:: sunpy.io.jp2
 
-.. _ioana:
+.. _io_ana:
 .. automodapi:: sunpy.io.ana
 
-Special File Readers
-====================
+.. _io_cdf:
+.. automodapi:: sunpy.io.cdf
 
-.. _iospecialgenx:
+.. _io_genx:
 .. automodapi:: sunpy.io.special.genx
-
-
-asdf (Advanced Scientific Data Format)
```

Comment:
To be restored elsewhere

Reply:
Is this restored elsewhere in this PR? If not maybe put it back in for us to avoid losing it between two PRs?